src/content/docs/extensions/vector.mdx

```diff
@@ -0,0 +1,266 @@
+```
+
+### Create the Vector Index
+
+Creating a new vector index as follows:
+```cypher
+CALL CREATE_VECTOR_INDEX(
+    'table_name',      // Name of the table containing the vector column
+    'index_name',      // Name to identify the vector index
+    'column_name',     // Name of the column containing vector embeddings
+    [option_name := option_value]  // Optional parameters for index configuration
+);
+```
+
+:::note
+For optional parameters, we use `:=` instead of `=` to specify values.
+:::
+
+We support following options during index creation.
+| option name | description | default |
+| ------------ | ------------ | ---------- |
+| mu                | max degree of nodes in the upper graph. | 30 |
+| ml                 | max degree of nodes in the lower graph. | 60 |
+| pu                | percentage of nodes sampled into the upper graph | 0.05 |
+| metric       | metric (distance computation) functions. supported are cosine, l2, l2sq, and dotproduct (dot_product). | cosine |
+| efc               | The number of candidate vertices to consider during the construction of the index. A higher value will result in a more accurate index, but will also increase the time it takes to build the index. | 200 |
+
+In our example, we create a vector index over the  `title_embedding` column from `Book` table.
+```cypher
+// Create a vector index on the title_embedding column
+CALL CREATE_VECTOR_INDEX('Book', 'title_vec_index', 'title_embedding');
+```
+
+### Query the Vector Index
+To perform similarity search using the vector index, use the `QUERY_VECTOR_INDEX` function:
+
+```cypher
+// Syntax for querying the index
+CALL QUERY_VECTOR_INDEX(
+    'table_name',      // Name of the table
+    'index_name',      // Name of the vector index
+    query_vector,      // Vector to search for
+    k,                 // Number of nearest neighbors to return
+    [option_name := option_value]  // Optional parameters
+) RETURN nn.id ORDER BY _distance;
```

Comment:
We should explain just below this example that we need to specify the `ORDER BY _distance` to get the results in ascending order of distance, and that if this clause is not specified, the results are not necessarily ordered by distance. Most other systems return vector results ordered by default, which is why we should be explicit here.